src/qml/pages/stub.qml

```diff
@@ -52,9 +52,21 @@ ApplicationWindow {
                 onClicked: indicator.progress = mouseX / width
             }
         }
-        ConnectionOptions {
-            Layout.preferredWidth: 400
-            focus: true
+        StackLayout {
```

Comment:
FYI stack doesn't support animation.

Reply:
Is it a blocker for this PR?

Reply:
No.

Reply:
yeah it's just used here for simplicity of the demo. #106 uses an uninteractive SwipeView which has the animations we want.